.github/workflows/spelling.yml

```
@@ -0,0 +1,30 @@
+name: Spell checking
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  pull_request_target:
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    name: Spell checker
+    runs-on: ubuntu-20.04
```

```suggestion
    runs-on: ubuntu-latest
```
I don't think we care about running on 20.04 specifically.